app/actions/network.js

```diff
@@ -0,0 +1,29 @@
+import { NETWORK_EXCEPTION } from '../types/message';
+
+export function fetch(type, data) {
```

Comment:
can these get named something different (possibly use the HTTP verbs)? Calling `fetch` with a different signature to `window.fetch` could lead to weird mistakes if you forget to import